patroni/__main__.py

```diff
@@ -61,18 +100,41 @@ def load_dynamic_configuration(self) -> None:
                 time.sleep(5)
 
     def get_tags(self) -> Dict[str, Any]:
+        """Get custom tags configured for this node, if any.
+
+        A custom tag is any tag added to the configuration ``tags`` section that is not one of ``clonefrom``,
+        ``nofailover``, ``noloadbalance`` or ``nosync``.
+
+        :returns: a dictionary of custom tags set for this node. The key is the tag name, and the value is the
+        corresponding tag value.
+        """
         return {tag: value for tag, value in self.config.get('tags', {}).items()
                 if tag not in ('clonefrom', 'nofailover', 'noloadbalance', 'nosync') or value}
 
     @property
     def nofailover(self) -> bool:
+        """``True`` if ``tags.nofailover`` configuration is enabled for this node, else ``False``."""
```

Comment:
```suggestion
        """:returns: ``True`` if ``tags.nofailover`` configuration is enabled for this node, else ``False``."""
```

Reply:
I'm not sure if you guys are going to eventually approve/merge https://github.com/zalando/patroni/pull/2699, but the documentation generated by `sphinx-build` based on docstrings will look weird if we start a docstring with `:returns:` as it expects a summary in the first line. It would generate an HTML with a content that looks like this when rendered in the browser:

```
property nofailover: bool
    True if tags.nofailover configuration is enabled for this node, else False.

    Type: returns
```

I initially thought of just using the same approach shown in the Python docs [property example](https://docs.python.org/3.6/library/functions.html#property:~:text=%40property%0A%20%20%20%20def%20voltage(self)%3A%0A%20%20%20%20%20%20%20%20%22%22%22Get%20the%20current%20voltage.%22%22%22%0A%20%20%20%20%20%20%20%20return%20self._voltage).

Note: we actually submitted a few commits to that other PR doing this same stuff for the files that already had docstrings merged into `master`.

What do you think?

Reply:
I see... I didn't check #2699 yet.